components/sponsored_benefits/app/models/sponsored_benefits/services/general_agency_manager.rb

```diff
@@ -61,6 +61,21 @@ def set_default_general_agency
         prev_default_ga_id = current_default_ga.id if current_default_ga
         broker_agency_profile.default_general_agency_profile = general_agency_profile
         broker_agency_profile.save!
+        employers = SponsoredBenefits::Organizations::PlanDesignOrganization.where(owner_profile_id: broker_agency_profile.id, has_active_broker_relationship: true)
+        employers.each do |employer|
+          if employer.general_agency_accounts.first.present?
+            employer.general_agency_accounts.first.benefit_sponsrship_general_agency_profile_id = general_agency_profile.id
+            employer.general_agency_accounts.first.save!
+          else
+            employer_ga_account = employer.general_agency_accounts.new
+            employer_ga_account.benefit_sponsrship_general_agency_profile_id = general_agency_profile.id
+            employer_ga_account.aasm_state = :active
+            employer_ga_account.start_on = Time.zone.now
+            employer_ga_account.broker_role_id = broker_agency_profile.primary_broker_role_id
+            employer_ga_account.benefit_sponsrship_broker_agency_profile_id = broker_agency_profile.id
+            employer_ga_account.save!
+          end
+        end
```

Comment:
we need to add a spec for this, this model already have tests, so we need to add a new one to handle this scenario